lib/vmpooler/api/v1.rb

```diff
@@ -645,6 +645,9 @@ def invalid_template_or_path(payload)
           result[params[:hostname]]['state'] = 'destroyed'
         elsif rdata['checkout']
           result[params[:hostname]]['running'] = ((Time.now - Time.parse(rdata['checkout'])) / 60 / 60).round(2)
+          result[params[:hostname]]['remaining'] = ((Time.parse(rdata['checkout']) + rdata['lifetime'].to_i*60*60 - Time.now) / 60 / 60).round(2).abs
```

Comment:
`abs` is used here to avoid a flaky test in `vm_spec.rb`. `remaining` would sometimes return as a negative.